WordPress/src/main/java/org/wordpress/android/util/CrashLoggingUtils.kt

```diff
@@ -33,7 +41,7 @@ class CrashLoggingUtils {
         }
 
         @JvmStatic fun stopCrashLogging() {
-            Sentry.clearContext()
+            Sentry.clearBreadcrumbs()
```

Comment:
I'm not sure we need to clear the breadcrumbs if we're closing down Sentry – in the older version clearing out the context would tear down Sentry completely so that we could re-init it later.

WDYT?

Reply:
That's a very good question. When I was looking for an alternative to `.clearContext()` I couldnt find any equivalent method in the v2.0.
Then, I checked what was doing inside to see if I could find a similar logic and I bumped into the implementation, which is calling this method :  
```

Reply:
Sounds great!